generate_index.py

```
@@ -86,27 +86,35 @@ def rewrite_html_content(self, content: str) -> str:
              # FontAwesome
-            ('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css', '/vendor/fontawesome/css/all.min.css'),
+            (r'https://cdnjs\.cloudflare\.com/ajax/libs/font-awesome/[^/]+/css/all\.min\.css', '/vendor/fontawesome/css/all.min.css'),
         ]
 
-        for old, new in replacements:
-            content = content.replace(old, new)
+        for pattern_str, new in replacements:
+            content = re.sub(pattern_str, new, content)
+
+        # Strip integrity and crossorigin attributes from tags referencing local /vendor/ files
+        def strip_sri(match):
```

Ruff (0.15.2)

[warning] Missing return type annotation for private function `strip_sri`

(ANN202)